src/Native/jitinterface/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(jitinterface)
+
+set(NATIVE_SOURCES
+    jitinterface.cpp
+    jitwrapper.cpp
+    corinfoexception.cpp
+)
+
+if (WIN32)
+    list(APPEND NATIVE_SOURCES jitinterface.def)
+    add_compile_options(/Zi)
```

Comment:
Moved PDB related settings from Native/jitinterface/CMakeLists.txt to Native/CMakeLists.txt.